docs/gatsby-node.js

```diff
@@ -83,13 +83,15 @@ function chooseTemplate(slug) {
 function searchNpm() {
   return new Promise((resolve, reject) => {
     childProcess.exec(
-      'npm search ps-design-system --json --parseable',
+      'npm search ps-design-system --parseable',
```

Comment:
--json is through errors for me. instead of using npm to format json - i've added a method to parse the tsv in code